PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# PR Details
+
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+
+<!--- Describe your changes in detail. Do not repeat the Issue description. -->
+
+## Related Issue
+
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
+<!--- Please link to the issue here: -->
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
+- [ ] Docs change / refactoring / dependency upgrade
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+## Checklist
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+
+- [ ] I have read the **CONTRIBUTING** document
+- [ ] My code follows the code style of this project
+- [ ] My change requires a change to the documentation 
+- [ ] I have updated the documentation accordingly (README.md for each UI component)
+- [ ] I have added tests to cover my changes (at least 1 spec for each UI component with the same coverage as the master branch)
+- [ ] All new and existing tests passed
+- [ ] A new branch needs to be created from master to evolve previous versions
+- [ ] Increase version in package.json following [Semantic Versioning](https://semver.org/)
```

Comment:
I am not sure I understand this point correctly. The idea is to increase the version in the branch, then? So, for every pull-request we will have a new versio (either major or minor)? If that is the case, we may have some problems when there are several branches pending to merge (all will have increased the number to the same value), and also, this will prevent generating versions which group several features/issues.

Reply:
It means that the master needs to be merged with the current, and then, increase the version following semantic versioning. 
It's true that we could face the case in which there's simultaneous branches and we need to update the version number before merging because another one did it before.

Reply:
Ok, you are right, when the master is merged to the current, it will be easy to check if there has been some change because of another branch, and at that point increase the version again